frappe/public/js/frappe/views/reports/report_utils.js

```diff
@@ -30,7 +30,12 @@ frappe.report_utils = {
 			colors: colors,
 			axisOptions: {
 				shortenYAxisNumbers: 1,
-				xAxisMode: 'tick'
+				xAxisMode: 'tick',
+				shortenNumberFunction : (label) => {
```

Comment:
Ignore duplication for now... identifying all the changes first and will figure out some better way later.